### ISSUE TITLE
New espeak variant: Pablo

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2704]
**Reported by lebenitez on 2012-10-08 18:33**
I have created an espeak variant called Pablo. It sounds very clear, and it may be added to NVDA.

**Comment by bhavyashah** [comment]
@feerrenrut Could we please have the variant file's attachment, merely out of interest? Regardless, variants should be ideally incorporated into ESpeak-NG itself and appear in NVDA as a consequence. Please request the folks at ESpeak-NG through its Github project located at https://github.com/espeak-ng/espeak-ng to integrate Pablo. Please add the NeedsExternalFix label to this ticket and/or close (as a wontfix?). @ehollig

**Comment by feerrenrut** [comment]
File has been added to https://github.com/nvaccess/nvda/issues/2704#issuecomment-155300961.
